candi/bashible/bundles/centos/bootstrap.sh.tpl

```diff
@@ -40,6 +40,13 @@ if [ "${VERSION_ID}" == "8" ] ; then
   alternatives --set python /usr/bin/python3
 fi
 
+for FS_NAME in `egrep -v -e '(^[[:space:]]*$|^#)' /etc/fstab | awk '{ print $1 }'`; do
+  if xfs_info $FS_NAME | grep -q ftype=0; then
```

Comment:
What we do if we do not have xfs_info binary installed ?

Reply:
I checked on centos 7/8/9 and rocky 9, looks like xfs_info exists in centos-like distros by default.
So, I didn't added installation of xfsprogs because of bootstrap.sh  size limitation (we must keep it smaller then 16384 bytes for cloud-init)